### DISCUSSION TITLE
How to render actions in custom Page content ?

### DISCUSSION THREAD

**Discussion by invaders-xx** [filamentphp/filament#4545] [Help] [answered]
I have a custom page where I want to render Actions within its content (not using getActions)
At the moment I build an array of Filament\Pages\Actions\Action and in the view I use:
```php
@foreach(actions as $action)
                <x-filament::pages.actions.action
                    :action="$action"
                    :label="$action->getLabel()"
                    component="filament::icon-button"
                    class="filament-page-icon-button-action"
                />
@endforeach
```
and I got an error:
Typed property Filament\Pages\Actions\Action::$livewire must not be accessed before initialization

Sure I am not doing the right way, but I don't know how to handle that.

Many thanks in advance for your help.

**Comment by danharrin** [comment, marked as answer]
Unfortunately actions can only be used in the header easily atm. This is a v3 feature when we introduce a new filament/actions package.

**Reply by invaders-xx** [reply]
Thanks @danharrin for your feedback. Still no expected date for V3 ?

**Reply by danharrin** [reply]
Nope, we have no Livewire release date yet.
